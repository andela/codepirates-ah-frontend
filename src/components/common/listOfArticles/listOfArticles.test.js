import React from 'react';
import { shallow } from 'enzyme';
import ListOfArticles from '.';
import articlesArr from '../../../__mocks__/mockedArticlesArray';

const renderComponent = (args) => {
  const props = { ...args };
  return shallow(<ListOfArticles {...props} />);
};

describe('List of articles', () => {
  it('render a list of articles component', () => {
    const component = renderComponent({
      articles: articlesArr,
      limitArticlesNumber: 4,
    });
    expect(component).toMatchSnapshot();
  });
});
