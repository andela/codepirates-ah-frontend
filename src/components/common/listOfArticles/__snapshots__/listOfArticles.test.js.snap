// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List of articles render a list of articles component 1`] = `ShallowWrapper {}`;
